edgelet/edgelet-docker/Cargo.toml

```diff
@@ -25,6 +24,7 @@ url_serde = "0.2"
 
 aziot-cert-common-http = { git = "https://github.com/Azure/iot-identity-service", branch = "main" }
 cert-client = { path = "../cert-client" }
+config-common = { git = "https://github.com/Azure/iot-identity-service", branch = "main" }
```

Comment:
in hindsight, we might have wanted to go with a less "general" name than `config-common`

Reply:
toml-config?

---
In reply to: [581281259](https://github.com/Azure/iotedge/pull/4465#discussion_r581281259) [](ancestors = 581281259)

Reply:
We should probably prefix everything with `aziot` at some point, at least for the sake of getting logs from `env_logger`.